storybook/stories/interactions/10a_brush_selection_bar_hist.story.tsx

```diff
@@ -22,7 +22,7 @@ export const Example = () => (
       onBrushEnd={action('onBrushEnd')}
       rotation={getChartRotationKnob()}
       roundHistogramBrushValues={boolean('roundHistogramBrushValues', false)}
-      allowBrushingLastHistogramBucket={boolean('allowBrushingLastHistogramBucket', false)}
+      allowBrushingLastHistogramBin={boolean('allowBrushingLastHistogramBin', false)}
```

Comment:
nit: maybe we make this the default value

```suggestion
      allowBrushingLastHistogramBin={boolean('allowBrushingLastHistogramBin', true)}
```

Reply:
Good catch thank you 9ebadbd